packages/core-components/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -76,6 +76,8 @@ export const MissingAnnotationEmptyState = ({ annotation }: Props) => {
             variant="contained"
             color="primary"
             href="https://backstage.io/docs/features/software-catalog/well-known-annotations"
+            target="_blank"
+            rel="noopener noreferrer"
```

Comment:
This should use the core `Link` instead!

So this could be

```
import { Link } from '@backstage/core-components';
....
          <Button
            variant="contained"
            color="primary"
            component={Link}
            to="https://backstage.io/docs/features/software-catalog/well-known-annotations"
```

Reply:
this way, the open-in-new-window-or-not behavior is handled centrally (and soon, configurably)

Reply:
@freben does this need a changeset?